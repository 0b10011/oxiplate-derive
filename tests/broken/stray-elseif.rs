use oxiplate_derive::Oxiplate;

#[derive(Oxiplate)]
#[oxiplate_inline = "{%- elseif should_do -%}{{ action }}"]
struct Data {
    should_do: bool,
    action: &'static str,
}

fn main() {
    let data = Data {
        should_do: true,
        action: "do something",
    };

    print!("{}", data);
}
