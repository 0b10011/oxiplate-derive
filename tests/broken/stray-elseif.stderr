error: Unexpected 'elseif' statement
 --> tests/broken/stray-elseif.rs:4:26
  |
4 | #[oxiplate_inline = "{%- elseif should_do -%}{{ action }}"]
  |                          ^^^^^^
